docs/index.rst

```diff
@@ -3,33 +3,109 @@
    You can adapt this file completely to your liking, but it should at least
    contain the root `toctree` directive.
 
-Welcome to django-graph-api's documentation!
+Django Graph API
 ============================================
+|travis| |gitter|
 
-**Django Graph API** is an implementation of GraphQL in Python, designed to work with the Django web framework.
+.. |gitter| image:: https://badges.gitter.im/django-graph-api/Lobby.svg
+   :alt: Join the chat at https://gitter.im/django-graph-api/Lobby
+   :target: https://gitter.im/django-graph-api/Lobby?utm_source=badge&utm_medium=badge&utm_campaign=pr-badge&utm_content=badge
+.. |travis| image:: https://travis-ci.org/melinath/django-graph-api.svg?branch=master
+   :alt: Build status on travis-ci
+   :target: https://travis-ci.org/melinath/django-graph-api
 
-The project began at DjangoCon US 2017.
+**Django Graph API** lets you quickly build GraphQL_ APIs in Python_. It is designed to work with the Django_ web framework.
 
-Why Not Graphene?
-++++++++++++++++++++++++++++++++++++++++++++
+What is GraphQL?
+----------------
 
-Graphene is the standard Python implmentation of GraphQL, and it is far more mature than Django Graph API. So why should you use this project instead of Graphene?
+GraphQL is an API query language created by Facebook in 2012 and open-sourced in 2015. Some of its benefits over REST are:
 
-* Django Graph API is Pythonic, while Graphene was ported over from Node.js and follows Javascript idioms.
+- getting the data you need and nothing more
+- getting nested fields without extra requests
+- strong typing
 
+For example, if you wanted to get your name and birthday, and the names and birthdays of all of your friends, you could query an API like this:
+::
 
+   POST http://myapp/graphql
+   "{
+       me {
+           name
+           birthday
+           friends {
+               name
+               birthday
+           }
+       }
+   }"
+
+And an example JSON response would be:
+::
+
+   {
+      "me": {
+         "name": "Buffy Summers",
+         "birthday": "1981-01-19",
+         "friends": [
+            {
+               "name": "Willow Rosenberg",
+               "birthday": "1981-08-01"
+            },
+            {
+               "name": "Xander Harris",
+               "birthday": null
+            }
+         ]
+      }
+   }
+
+For an full introduction to GraphQL, you can read the `official documentation`_.
+
+If you have a Github account, you can try out their `GraphQL API explorer`_.
+
+Why Django Graph API?
+---------------------
+
+We see GraphQL as a promising alternative to REST.
+
+In order to increase its usage amongst Python developers, we are trying to create a library that stays up to date with the GraphQL specs and that embraces all of the things we love about Python:
+
+- simple, readable, and elegant
+- great documentation
+- supportive open-source community
```

Comment:
Would it make sense to say something here about the goal to make it extensible / support common use cases out of the box or is that too much / too early?

Reply:
I originally had something like that in but it felt too early. We should add it though once we supported more of the core features.